### PR TITLE
Add vtol-specific information to the airframe doc

### PR DESCRIPTION
**Pull request by hamishwillee**
The VTOL information was in an airframe overview doc. It probably belongs here.

### CODE REVIEW

File: en/airframes/adding_a_new_frame.md [hunk 1]
```diff
@@ -162,8 +162,21 @@ S: 0 3      0  20000 -10000 -10000  10000
...
+* `VT_ARSP_TRANS` is the airspeed in m/s at which the plane transitions into forward flight. Setting it too low can cause a stall during the transition.
+* `RC_MAP_TRANS_SW` should be assigned to a RC switch before flight. This allows you to check if the multicopter- and fixed wing mode work properly. (Can also be used to switch manually between those two control modes during flight)
+
+
+## Getting a New Airframe to Show in QGroundControl
 
 The airframe meta data is bundled in the .px4 firmware file (which is a zipped JSON file).
 
-> **Note** Ensure to flash the resulting .px4 file in QGroundControl (custom file option) to load the meta data into the application. The new airframe will then be available in the user interface once you restart QGroundControl.
+> **Note** Flash the resulting **.px4** file in *QGroundControl* (custom file option) to load the meta data into the application. The new airframe will then be available in the user interface once you restart *QGroundControl*.
```
Comment (hamishwillee) [inline en/airframes/adding_a_new_frame.md:199]: @bkueng I think this means "As soon as you have loaded the custom firmware onto any target vehicle the metadata will be available in the QGC airframe section. Can you confirm?

File: en/airframes/adding_a_new_frame.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+# Adding a New Airframe Configuration
 
 PX4 uses canned configurations as starting point for airframes. Adding a configuration is straightforward: Create a new file which is prepended with a free autostart ID in the [init.d folder](https://github.com/PX4/Firmware/tree/master/ROMFS/px4fmu_common/init.d) and [build and upload](../setup/building_px4.md) the software.
 
 Developers not wanting to create their own configuration can instead customize existing configurations using text files on the microSD card, as detailed on the [custom system startup](../advanced/system_startup.md) page.
 
-## Airframe configurations
+## Airframe Configurations
 
 An airframe configuration consists of three main blocks:
```
Comment (hamishwillee) [inline en/airframes/adding_a_new_frame.md]: @bkueng The third block is "tuning gains" (not shown here). We don't have any comment/section on that block (while we do on config file and mixer file blocks) - where is that set/specified. Are there any other documents we  should link to explaining how to set these gains?

File: en/airframes/adding_a_new_frame.md [hunk 3]
```diff
@@ -162,8 +162,21 @@ S: 0 3      0  20000 -10000 -10000  10000
 
 ```
 
-### Getting the new airframe to show in QGroundControl
+
+
+## VTOL-specific Configuration
```
Comment (hamishwillee) [inline en/airframes/adding_a_new_frame.md]: Does this section make sense here? Is there a better place?

File: en/airframes/adding_a_new_frame.md (hunk 2 above)
Comment (bkueng) [reply en/airframes/adding_a_new_frame.md]: It's this block:
```
	param set BAT_N_CELLS 2
	param set FW_AIRSPD_MAX 15
	param set FW_AIRSPD_MIN 10
	param set FW_AIRSPD_TRIM 13
	param set FW_R_TC 0.3
	param set FW_P_TC 0.3
	param set FW_L1_DAMPING 0.74
	param set FW_L1_PERIOD 16
	param set FW_LND_ANG 15
	param set FW_LND_FLALT 5
	param set FW_LND_HHDIST 15
	param set FW_LND_HVIRT 13
	param set FW_LND_TLALT 5
	param set FW_THR_LND_MAX 0
	param set FW_PR_FF 0.35
	param set FW_RR_FF 0.6
	param set FW_RR_P 0.04
```
And it's more than just tuning gains. It includes vehicle-specific parameter settings, for example the number of battery cells, or things like the gps position offset relative to the board. I think it makes sense to add a short section about that here and include a link to the tuning gains page.

For the mixer we can link to https://dev.px4.io/en/concept/mixing.html.

File: en/airframes/adding_a_new_frame.md (hunk 1 above)
Comment (bkueng) [reply en/airframes/adding_a_new_frame.md:199]: I think so, but I never load FW that way, so I don't really know how that works :)

File: en/airframes/adding_a_new_frame.md (hunk 3 above)
Comment (bkueng) [reply en/airframes/adding_a_new_frame.md]: Yes, looks good. Unless there is a VTOL-specific tuning page?

Comment (hamishwillee) [reply en/airframes/adding_a_new_frame.md]: Good point thanks. I'll add a link to the tuning docs.

File: en/airframes/adding_a_new_frame.md (hunk 2 above)
Comment (hamishwillee) [reply en/airframes/adding_a_new_frame.md]: Absolutely.

File: en/airframes/adding_a_new_frame.md [hunk 4]
```diff
@@ -63,21 +67,34 @@ then
...
 set MAV_TYPE 1
+```
+
+Set the mixer:
+```bash
 # Set mixer
 set MIXER wingwing
+```
+
+```bash
```
Comment (hamishwillee) [inline en/airframes/adding_a_new_frame.md]: @bkueng Is there anything we can say about this section below? (ie what is happening here)

File: en/airframes/adding_a_new_frame.md [hunk 5]
```diff
@@ -1,25 +1,26 @@
...
 PX4 uses canned configurations as starting point for airframes. Adding a configuration is straightforward: Create a new file which is prepended with a free autostart ID in the [init.d folder](https://github.com/PX4/Firmware/tree/master/ROMFS/px4fmu_common/init.d) and [build and upload](../setup/building_px4.md) the software.
 
 Developers not wanting to create their own configuration can instead customize existing configurations using text files on the microSD card, as detailed on the [custom system startup](../advanced/system_startup.md) page.
 
-## Airframe configurations
+## Airframe Configurations
 
 An airframe configuration consists of three main blocks:
 
   * The apps it should start, e.g. multicopter or fixed wing controllers
```
Comment (hamishwillee) [inline en/airframes/adding_a_new_frame.md]: @bkueng This is just the bit ` set MAV_TYPE 1` right? Any other "apps" are automatically set from whatever number you enter? 
- Is there a list of numbers
- should we call this "The controllers that should be started, e.g.. ..." instead?
- Perhaps move this below to match order of declaration in config file?

Also, should we add "documentation" in this list - it is important!

File: en/airframes/adding_a_new_frame.md [hunk 6]
```diff
@@ -1,25 +1,26 @@
...
-  * The physical configuration of the system (e.g. a plane, wing or multicopter). This is called mixer.
```
Comment (hamishwillee) [inline en/airframes/adding_a_new_frame.md]: @bkueng Just for my info, this is the mixer file - which you specify in the config file using `set MIXER wingwing` right?

**Comment by hamishwillee** [comment]
Hi @bkueng 

Thanks for your review. As you suggested I added link to the mixer concepts doc and to tuning section (and clarified that parameters are not just for tuning). The tuning section removed the "specific content" I added previously, and instead now links to our tuning guides for each frame. 

In addition, I split the example file out so that the content of each section is separately described - docs, setting mixer etc. I think that makes things more clear. This is worth your re-review please.

There are a couple of questions inline too.

File: en/airframes/adding_a_new_frame.md (hunk 5 above)
Comment (bkueng) [reply en/airframes/adding_a_new_frame.md]: For this example it's just `MAV_TYPE`, yes.
- List of numbers: it's the `MAV_TYPE` param and @dagar added the full param description for the numbers in a PR: https://github.com/PX4/Firmware/pull/8001/files#diff-86d388773b907f1fd874d2990d901c8dR79 (It matches the definition in mavlink: http://mavlink.org/messages/common).
- It's more than controllers. For example it's also the land detector that depends on the type. And the Intel Aero starts a specific ESC driver: https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/4070_aerofc#L71
- Makes sense

File: en/airframes/adding_a_new_frame.md (hunk 6 above)
Comment (bkueng) [reply en/airframes/adding_a_new_frame.md]: Right

File: en/airframes/adding_a_new_frame.md (hunk 4 above)
Comment (bkueng) [reply en/airframes/adding_a_new_frame.md]: We can say that this configures the PWM outputs: which outputs we want to drive/activate, and the levels.

**Comment by hamishwillee** [comment]
Thanks @bkueng - I integrated your information, improved the structure, merged.

**Pull request merged**